### PR TITLE
LPS-48334

### PR DESCRIPTION
**Pull request by blzaugg**

### CODE REVIEW

File: readme/7.0/BREAKING_CHANGES.markdown [hunk 1]
```diff
@@ -73,6 +73,37 @@ in ascending chronological order.
...
+`editor.wysiwyg.portal-web.docroot.html.portlet.message_boards.edit_message.bb_code.jsp`
+
+#### Why was this change made?
+
+Since Liferay Frontend Editor BBCode Web has been depricated since 6.1, it was
+time to remove it completely. This frees up developmnet and support resources to
+focus on supported features.
+
+---------------------------------------
+
```
Comment (blzaugg) [inline readme/7.0/BREAKING_CHANGES.markdown]: /cc @jhinkey, @sez11a

Comment (jhinkey) [inline readme/7.0/BREAKING_CHANGES.markdown]: /cc @codyhoag

**Comment by jbalsas** [comment]
ci:retest

**Comment by jonmak08** [comment]
Merged and pushed.  Thanks!

**Comment by jonmak08** [comment]
Merged and pushed.  Thanks!

**Comment by jonmak08** [comment]
Merged and pushed.  Thanks!

**Comment by jonmak08** [comment]
Merged and pushed.  Thanks!

**Comment by jonmak08** [comment]
Merged and pushed.  Thanks!

**Comment by jonmak08** [comment]
Merged and pushed.  Thanks!

**Pull request closed without merging**